app/models/provider.rb

```diff
@@ -75,6 +75,16 @@ def name_and_code
     "#{name} (#{code})"
   end
 
+  def trainees_to_be_placed
```

Comment:
I'm not sure about the naming of this query. It's a bit ambiguous in terms of "where" they need to be placed. Something `without required placements` is a little clearer in what the query is retrieving